webapp/channels/src/components/delete_channel_modal/delete_channel_modal.tsx

```diff
@@ -77,19 +77,24 @@ export default class DeleteChannelModal extends React.PureComponent<Props, State
                 <Modal.Body>
                     <div className='alert alert-danger'>
                         {!canViewArchivedChannels &&
-                            <FormattedMarkdownMessage
-                                id='delete_channel.question'
-                                defaultMessage='This will archive the channel from the team and remove it from the user interface. Archived channels can be unarchived if needed again. \n \nAre you sure you wish to archive the {display_name} channel?'
+                            <FormattedMessage
+                                id='deleteChannelModal.cannotViewArchivedChannelsWarning'
+                                defaultMessage='This will archive the channel from the team and remove it from the user interface. Archived channels can be unarchived if needed again.{br}{br}Are you sure you wish to archive the <b>{display_name}</b> channel?'
                                 values={{
                                     display_name: this.props.channel.display_name,
+                                    br: <br/>,
```

Comment:
New line replacement

Reply:
Could you actually split these into two separate `FormattedMessage`s in their own `p` tags? They can actually reuse the "Are you sure you wish to archive the X channel?" string as well. We generally shouldn't need to use `br` tags for things like this

Reply:
thanks that's a great suggestion